src/admin/localService/loadFile.test.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import {loadFile} from "./loadFile";
+import fs from "fs";
+
+describe("src/admin/localService/loadFile", () => {
+  beforeAll(async () => {
+    // Arrange
+    await fs.promises.mkdir("./test/dir", {recursive: true});
+    await fs.promises.writeFile("./test/dir/some-file.txt", "found me!");
+  });
+  it("finds and reads a file", async () => {
+    // Act
+    const filePath = await loadFile("test", "some-file.txt");
+    // Assert
+    expect(filePath).toEqual("found me!");
+  });
+  it("throws an error if no file is found in an existing directory", async () => {
+    // Act
+    return expect(loadFile("test", "missing-file.txt")).rejects.toThrow(
+      // Assert
+      "missing-file.txt not found. Please enter a full cred repo"
+    );
+  });
+  it("throws an error if the entered directory doesn't exist", async () => {
+    //arrange
+    expect.assertions(1);
```

Comment:
what does this do exactly, I assume it ensures that a certain number of assertions are made? "arrange" isn't that helpful as a comment IMO, if anything it should just clarify what assertions is for

Reply:
> what does this do exactly, I assume it ensures that a certain number of assertions are made?

Yes, it ensures the test actually runs. Basically, if the assertion never executes due to some issue where the promise never resolves, we don't want a silent failure where jest just accepts the test as passing because nothing threw. I'm setting these for the two other tests for the sake of consistency.

> "arrange" isn't that helpful as a comment IMO, if anything it should just clarify what assertions is for

"arrange, act, assert (AAA)" is a simple organizing structure for setting up tests. I think setting the expected assertion quantity falls under the "arrange" step since it's testing instrumentation. I'm happy to remove these comments but I find them helpful.